src/localization/locales/en-US.json

```diff
@@ -112,6 +113,26 @@
       "home_card_kobo_title": "KoBoToolbox",
       "home_card_kobo_link": "Learn more about KoBoToolbox features"
     },
+    "tools": {
+      "kobo": {
+        "title": "KoBoToolbox",
+        "url": "https://www.kobotoolbox.org/",
+        "img": {
```

Comment:
question: why image attributes need to be on the file with data to be translated?

Reply:
thought: I am also not sure about this, but potentially localization is also related to how content displays, because on different locales depending on the content the image can use more or less space, not having another places to handle something like this related to each locale, I think this might be ok for now. My only concern is that translators might change this values breaking the application, but maybe we are small enough that this will not be an issue

Reply:
My thinking is the same as Jose's:
* The image in a different language could be a different size
* There isn't another good place to store the data; putting it in two places will make future updating more difficult.